src/lib/routing-table.js

```diff
@@ -102,4 +124,12 @@ class RoutingTable {
   }
 }
 
+function getPrefixLength (str1, str2) {
```

Comment:
This function feels dangerous, because if for instance `str1 = 'g.michiel.'` and `str2= 'g.michael.'`, it would return `'g.mich'.length` instead of `'g.'.length`